### ISSUE TITLE
Dynamic objects

### ISSUE THREAD

**Issue by bhack** [zju3dv/DetectorFreeSfM#30]
Does this require (or support) masks for dynamic objects filtering?

**Comment by hxy-123** [comment]
No, the dynamic object is not considered in the system, because the RANSAC will filter most of the matches on the dynamic objects as long as the dynamic part does not occupy the domain content of images. However, I think adding the mask of dynamic objects can further boost the performance, which can be straightforwardly achieved by mask coarse level matches with the mask :).

**Comment by bhack** [comment]
Let me know if you will add this option for dominant dynamic objects.
